src/Microsoft.OData.Edm/ExtensionMethods/ExtensionMethods.cs

```diff
@@ -100,6 +98,31 @@ public static IEnumerable<IEdmOperation> FindBoundOperations(this IEdmModel mode
             return FindAcrossModels(model, bindingType, findBoundOperations, mergeFunctions);  // search built-in EdmCoreModel and CoreVocabularyModel.
         }
 
+        /// Searches for bound operations based on the binding type, across models, returns an empty enumerable if no operation exists.
+        /// </summary>
+        /// <param name="model">The model to search.</param>
+        /// <param name="bindingType">Type of the binding.</param>
+        /// <returns>A set of operations that share the binding type or empty enumerable if no such operation exists.</returns>
+        public static IEnumerable<IEdmOperation> FindDeclaredBoundOperationsAcrossModels(this IEdmModel model, IEdmType bindingType)
+        {
+            IList<IEdmOperation> bindableOperations = new List<IEdmOperation>();
+
+            foreach (IEdmOperation operation in model.FindDeclaredBoundOperations(bindingType))
+            {
+                bindableOperations.Add(operation);
+            }
+
+            foreach (IEdmModel reference in model.ReferencedModels)
+            {
+                foreach (IEdmOperation operation in reference.FindDeclaredBoundOperations(bindingType))
+                {
+                    bindableOperations.Add(operation);
+                }
+            }
+
+            return bindableOperations;

```

Comment:
This might be a good candidate for `yield return` as well

Reply:
+1